deps/SnoopCompile/snoop_bot_config.jl

```diff
@@ -3,5 +3,5 @@ using CompileBot
 botconfig = BotConfig(
     "Plots",
     version = [v"1.6", v"1.7"],  # <<< keep versions in sync with .github/workflows/SnoopCompile.yml
-    # else_version = v"1.8",
+    else_version = "nightly",
```

Comment:
Would this mean, that people running julia 1.2 get the nightly precompile statements? If so, we should add `nightly` to `version`.

Reply:
Ha, that's a good catch.